src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackend.java

```diff
@@ -37,6 +45,12 @@
  */
 public class MongoBackend {
     
+    /**
+     * Available data models when persisting data at Mongo.
```

Comment:
Nice catch :) Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/1ee8c25c211966770b915afcf63c31ac31bdeac8